android/app/src/main/java/org/fossasia/openevent/fragments/AboutFragment.java

```diff
@@ -176,5 +218,92 @@ private void setUpCustomTab(String url) {
 
         customTabsIntent.launchUrl(getActivity(), uri);
     }
+
+    @Override
+    public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
+        super.onCreateOptionsMenu(menu, inflater);
+
+        inflater.inflate(R.menu.menu_home, menu);
+        // Get the SearchView and set the searchable configuration
+        SearchManager searchManager = (SearchManager)getContext(). getSystemService(Context.SEARCH_SERVICE);
+        SearchView searchView = (SearchView) menu.findItem(R.id.action_search_home).getActionView();
+        // Assumes current activity is the searchable activity
+        searchView.setSearchableInfo(searchManager.getSearchableInfo(getActivity().getComponentName()));
+        searchView.setIconifiedByDefault(true); // Do not iconify the widget; expand it by default
+    }
+
+    @Override
+    public void onResume() {
+        super.onResume();
+        if (sessionsListAdapter != null) {
```

Comment:
Instead of using 
```Java
if(blah != null) {
    doSomething();
    if(blahblah != null) {
        doSomethingElse();
    }
}
```

Use this

```Java
if(blah == null)
    return;

doSomething();

if(blahblah == null)
    return;

doSomethingElse();
```

Avoid nesting